main.py

```diff
@@ -0,0 +1,47 @@
+import random;
+names = []
+#A user-defined function that collects names from the user to create a shortlist and verifies user input
+def namePicker():
```

Comment:
Function and variables names should be lowercase, with words separated by underscores as necessary.